mhkit/tests/test_wave.py

```diff
@@ -770,6 +770,83 @@ def test_read_wecSim_with_moorDyn(self):
         self.assertEqual(len(ws_output['moorDyn']),7)
         self.assertEqual(len(ws_output['ptosim']),0)
 
+class TestWPTOhindcast(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+
+        self.my_swh = pd.read_csv(join(datadir,'hindcast/multi_year_hindcast.csv'),index_col = 'time_index',
+        names = ['time_index','significant_wave_height_0'],header = 0, 
+        dtype = {'significant_wave_height_0':'float32'})
+        self.my_swh.index = pd.to_datetime(self.my_swh.index)
+
+        self.ml = pd.read_csv(join(datadir,'hindcast/single_year_hindcast_multiloc.csv'),index_col = 'time_index',
+        names = ['time_index','mean_absolute_period_0','mean_absolute_period_1'],
+        header = 0, dtype = {'mean_absolute_period_0':'float32',
+        'mean_absolute_period_1':'float32'})
+        self.ml.index = pd.to_datetime(self.ml.index)
+
+        self.mp = pd.read_csv(join(datadir,'hindcast/multiparm.csv'),index_col = 'time_index',
+        names = ['time_index','energy_period_0','mean_zero-crossing_period_0'],
+        header = 0, dtype = {'energy_period_0':'float32',
+        'mean_zero-crossing_period_0':'float32'})
+        self.mp.index = pd.to_datetime(self.mp.index)
+
+        self.ml_meta = pd.read_csv(join(datadir,'hindcast/multiloc_meta.csv'),index_col = 0,
+        names = [None,'water_depth','latitude','longitude','distance_to_shore','timezone'
+        ,'jurisdiction'],header = 0, dtype = {'water_depth':'float32','latitude':'float32'
+        ,'longitude':'float32','distance_to_shore':'float32','timezone':'int16'})
+
+        self.metadata = pd.read_csv(join(datadir,'hindcast/meta.csv'),index_col = 0,
```

Comment:
Rebecca could you just change this from `metadata` to something like `my_meta` to be consistent. And also change the data filename to include the multi-year in the name as well>